docs/error-messages/compiler-errors-1/compiler-error-c2289.md

```diff
@@ -1,18 +1,21 @@
 ---
-description: "Learn more about: Compiler Error C2289"
 title: "Compiler Error C2289"
-ms.date: "11/04/2016"
+description: "Learn more about: Compiler Error C2289"
+ms.date: 11/04/2016
 f1_keywords: ["C2289"]
 helpviewer_keywords: ["C2289"]
-ms.assetid: cb41a29e-1b06-47dc-bfce-8d73bd63a0df
 ---
 # Compiler Error C2289
 
-same type qualifier used more than once
+> same type qualifier used more than once
+
+## Remarks
 
 A type declaration or definition uses a type qualifier (**`const`**, **`volatile`**, **`signed`**, or **`unsigned`**) more than once, causing an error under ANSI compatibility (**/Za**).
 
-The following sample generates C2286:
+## Example
+
+The following example generates C2286:
```

Comment:
Should C2286 be C2289?

Reply:
yes - good catch (pre-existing). @rageking8, though this error case needs to be updated because it doesn't generate that error anyway. Let's get this through (I'll fix the comment errors) but we need to address the C2274 error example because the compiler no longer behaves that way.